applications/nrf5340_audio/src/bluetooth/bt_content_control/content/bt_media_control.h

```diff
@@ -53,21 +57,20 @@ int ble_mcs_play_pause(struct bt_conn *conn);
  *
  * @return 0 for success, error otherwise.
  */
-int ble_mcp_conn_disconnected(struct bt_conn *conn);
-
+int bt_media_control_conn_disconnected(struct bt_conn *conn);
 
 /**
  * @brief  Initialize the Media Control Client.
  *
  * @return 0 for success, error otherwise.
  */
-int ble_mcs_client_init(void);
+int bt_media_control_client_init(void);
 
 /**
  * @brief  Initialize the Media Control Server.
  *
  * @return 0 for success, error otherwise.
  */
-int ble_mcs_server_init(ble_mcs_play_pause_cb le_audio_play_pause_cb);
+int bt_media_control_server_init(bt_media_control_play_pause_cb play_pause_cb);
```

Comment:
`play_pause_cb` is undocumented. Are these header files built for documentation, or just internal?

Reply:
These are internal, but I will document the play_pause_cb